tests/sources/variadic.elpi

```diff
@@ -0,0 +1,5 @@
+func f int, int -> int.. .
+f N D R :- R is N div D.
+f N D R M :- f N D R, M is N mod D.
```

Comment:
this type checks fine, runs fine, but det check fails with an assertion

Reply:
it now works https://github.com/LPCIC/elpi/pull/348/commits/0215834c2550ef36433f492ecb5176b96aef80d0 but should somehow not require the cut